Cannot find module 'debug/browser' when running 'npm run watch' for lighthouse-extension

I just sync the latest code, navigate to `lighthouse-extension` folder and run `npm install` (both under the root path and lighthouse-extensions path) and `npm run watch`, the following errors come, maybe a global environment issue or what else do I need to do?

```
E:\Personal\lighthouse\lighthouse-extension>npm run watch

> lighthouse@ watch E:\Personal\lighthouse\lighthouse-extension
> gulp watch

[14:51:31] Using gulpfile E:\Personal\lighthouse\lighthouse-extension\gulpfile.js
[14:51:31] Starting 'lint'...
[14:51:31] Starting 'browserify'...
[14:51:31] Starting 'browserify-lighthouse'...
[14:51:31] Starting 'html'...
[14:51:31] Finished 'html' after 764 ms
[14:51:32] Finished 'lint' after 1 s
stream.js:74
      throw er; // Unhandled stream error in pipe.
      ^

Error: Cannot find module 'debug/browser' from 'E:\Personal\lighthouse\lighthouse-core\lib'
    at E:\Personal\lighthouse\lighthouse-extension\node_modules\browser-resolve\node_modules\resolve\lib\async.js:46:17
    at process (E:\Personal\lighthouse\lighthouse-extension\node_modules\browser-resolve\node_modules\resolve\lib\async.js:173:43)
    at ondir (E:\Personal\lighthouse\lighthouse-extension\node_modules\browser-resolve\node_modules\resolve\lib\async.js:188:17)
    at load (E:\Personal\lighthouse\lighthouse-extension\node_modules\browser-resolve\node_modules\resolve\lib\async.js:69:43)
    at onex (E:\Personal\lighthouse\lighthouse-extension\node_modules\browser-resolve\node_modules\resolve\lib\async.js:92:31)
    at E:\Personal\lighthouse\lighthouse-extension\node_modules\browser-resolve\node_modules\resolve\lib\async.js:22:47
    at FSReqWrap.oncomplete (fs.js:123:15)
```

--- a/lighthouse-cli/chrome-launcher.ts
+++ b/lighthouse-cli/chrome-launcher.ts
@@ -29,6 +29,7 @@ const rimraf = require('rimraf');
 const log = require('../lighthouse-core/lib/log');
 const spawn = childProcess.spawn;
 const execSync = childProcess.execSync;
+const os = require('os');
 
 class ChromeLauncher {
   prepared: Boolean = false
@@ -191,7 +192,10 @@ class ChromeLauncher {
         launcher
           .isDebuggerReady()
           .then(() => {
-            log.log('ChromeLauncher', waitStatus + `${green}✓${reset}`);
+            if (os.platform() == 'win32')
+              log.log('ChromeLauncher', waitStatus + `${green}\u221A${reset}`);
+            else
+              log.log('ChromeLauncher', waitStatus + `${green}✓${reset}`);
             resolve();
           })
           .catch(err => {

--- a/lighthouse-cli/chrome-launcher.ts
+++ b/lighthouse-cli/chrome-launcher.ts
@@ -29,7 +29,14 @@ const rimraf = require('rimraf');
 const log = require('../lighthouse-core/lib/log');
 const spawn = childProcess.spawn;
 const execSync = childProcess.execSync;
-const os = require('os');
+
+const green = '\x1B[32m';
+const reset = '\x1B[0m';
+
+const isWindows = process.platform === 'win32';
+
+// See https://github.com/GoogleChrome/lighthouse/issues/1228
+const tick = isWindows ? '\u221A' : '✓';
 
 class ChromeLauncher {
   prepared: Boolean = false
@@ -179,9 +186,6 @@ class ChromeLauncher {
       let retries = 0;
       let waitStatus = 'Waiting for browser.';
       (function poll() {
-        const green = '\x1B[32m';
-        const reset = '\x1B[0m';
-
         if (retries === 0) {
           log.log('ChromeLauncher', waitStatus);
         }
@@ -192,10 +196,7 @@ class ChromeLauncher {
         launcher
           .isDebuggerReady()
           .then(() => {
-            if (os.platform() == 'win32')
-              log.log('ChromeLauncher', waitStatus + `${green}\u221A${reset}`);
-            else
-              log.log('ChromeLauncher', waitStatus + `${green}✓${reset}`);
+            log.log('ChromeLauncher', waitStatus + `${green}${tick}${reset}`);
             resolve();
           })
           .catch(err => {
@@ -217,7 +218,7 @@ class ChromeLauncher {
         });
 
         log.log('ChromeLauncher', 'Killing all Chrome Instances');
-        if (process.platform === 'win32') {
+        if (isWindows) {
           execSync(`taskkill /pid ${this.chrome.pid} /T /F`);
         } else {
           process.kill(-this.chrome.pid);

--- a/lighthouse-cli/printer.ts
+++ b/lighthouse-cli/printer.ts
@@ -32,7 +32,21 @@ const fs = require('fs');
 const ReportGenerator = require('../lighthouse-core/report/report-generator');
 const Formatter = require('../lighthouse-core/formatters/formatter');
 const log = require('../lighthouse-core/lib/log');
-const os = require('os');
+
+const green = '\x1B[32m';
+const red = '\x1B[31m';
+const yellow = '\x1b[33m';
+const purple = '\x1b[95m';
+const reset = '\x1B[0m';
+const bold = '\x1b[1m';
+
+const isWindows = process.platform === 'win32';
+
+// See https://github.com/GoogleChrome/lighthouse/issues/1228
+const tick = isWindows ? '\u221A' : '✓';
+const cross = isWindows ? '\u00D7' : '✘';
+const whiteSmallSquare = isWindows ? '\u0387' : '▫';
+const doubleLightHorizontal = '──';
 
 /**
  * Verify output path to use, either stdout or a file path.
@@ -50,22 +64,8 @@ function formatAggregationResultItem(score: boolean | number | string, suffix?: 
   // Until we only support node 6 we can not use default args.
   suffix = suffix || '';
 
-  const green = '\x1B[32m';
-  const red = '\x1B[31m';
-  const yellow = '\x1b[33m';
-  const purple = '\x1b[95m';
-  const reset = '\x1B[0m';
-
   if (typeof score === 'boolean') {
-    if (os.platform() == 'win32') {
-      const check = `${green}\u221A${reset}`;
-      const fail = `${red}\u00D7${reset}`;
-      return score ? check : fail;
-    } else {
-      const check = `${green}✓${reset}`;
-      const fail = `${red}✘${reset}`;
-      return score ? check : fail;
-    }
+    return score ? `${green}${tick}${reset}` : `${red}${cross}${reset}`;
   }
   if (typeof score !== 'number') {
     return `${purple}${score}${reset}`;
@@ -98,20 +98,12 @@ function createOutput(results: Results, outputMode: OutputMode): string {
   }
 
   // Pretty printed.
-  const bold = '\x1b[1m';
-  const reset = '\x1B[0m';
   const version = results.lighthouseVersion;
-
   let output = `\n\n${bold}Lighthouse (${version}) results:${reset} ${results.url}\n\n`;
 
   results.aggregations.forEach(aggregation => {
     const total = aggregation.total ? ': ' + formatAggregationResultItem(Math.round(aggregation.total * 100), '%') : '';
-
-    if (os.platform() == 'win32') {
-      output += `\u0387 ${bold}${aggregation.name}${reset}${total}\n\n`;
-    } else {
-      output += `▫ ${bold}${aggregation.name}${reset}${total}\n\n`;
-    }
+    output += `${whiteSmallSquare} ${bold}${aggregation.name}${reset}${total}\n\n`;
 
     aggregation.score.forEach(item => {
       const score = (item.overall * 100).toFixed(0);
@@ -132,7 +124,7 @@ function createOutput(results: Results, outputMode: OutputMode): string {
         if (auditResult.comingSoon === true)
           return;
 
-        let lineItem = ` ── ${formatAggregationResultItem(auditResult.score)} ${auditResult.description}`;
+        let lineItem = ` ${doubleLightHorizontal} ${formatAggregationResultItem(auditResult.score)} ${auditResult.description}`;
         if (auditResult.displayValue) {
           lineItem += ` (${bold}${auditResult.displayValue}${reset})`;
         }

--- a/lighthouse-cli/printer.ts
+++ b/lighthouse-cli/printer.ts
@@ -32,6 +32,7 @@ const fs = require('fs');
 const ReportGenerator = require('../lighthouse-core/report/report-generator');
 const Formatter = require('../lighthouse-core/formatters/formatter');
 const log = require('../lighthouse-core/lib/log');
+const os = require('os');
 
 /**
  * Verify output path to use, either stdout or a file path.
@@ -56,9 +57,15 @@ function formatAggregationResultItem(score: boolean | number | string, suffix?: 
   const reset = '\x1B[0m';
 
   if (typeof score === 'boolean') {
-    const check = `${green}✓${reset}`;
-    const fail = `${red}✘${reset}`;
-    return score ? check : fail;
+    if (os.platform() == 'win32') {
+      const check = `${green}\u221A${reset}`;
+      const fail = `${red}\u00D7${reset}`;
+      return score ? check : fail;
+    } else {
+      const check = `${green}✓${reset}`;
+      const fail = `${red}✘${reset}`;
+      return score ? check : fail;
+    }
   }
   if (typeof score !== 'number') {
     return `${purple}${score}${reset}`;
@@ -94,11 +101,17 @@ function createOutput(results: Results, outputMode: OutputMode): string {
   const bold = '\x1b[1m';
   const reset = '\x1B[0m';
   const version = results.lighthouseVersion;
+
   let output = `\n\n${bold}Lighthouse (${version}) results:${reset} ${results.url}\n\n`;
 
   results.aggregations.forEach(aggregation => {
-		const total = aggregation.total ? ': ' + formatAggregationResultItem(Math.round(aggregation.total * 100), '%') : ''
-    output += `▫ ${bold}${aggregation.name}${reset}${total}\n\n`;
+    const total = aggregation.total ? ': ' + formatAggregationResultItem(Math.round(aggregation.total * 100), '%') : '';
+
+    if (os.platform() == 'win32') {
+      output += `\u0387 ${bold}${aggregation.name}${reset}${total}\n\n`;
+    } else {
+      output += `▫ ${bold}${aggregation.name}${reset}${total}\n\n`;
+    }
 
     aggregation.score.forEach(item => {
       const score = (item.overall * 100).toFixed(0);

--- a/lighthouse-core/formatters/critical-request-chains.js
+++ b/lighthouse-core/formatters/critical-request-chains.js
@@ -22,7 +22,17 @@ const path = require('path');
 const fs = require('fs');
 const Formatter = require('./formatter');
 const html = fs.readFileSync(path.join(__dirname, 'partials/critical-request-chains.html'), 'utf8');
-const os = require('os');
+
+const isWindows = process.platform === 'win32';
+
+// See https://github.com/GoogleChrome/lighthouse/issues/1228
+const heavyHorizontal = isWindows ? '\u2500' : '━';
+const heavyVertical = isWindows ? '\u2502 ' : '┃ ';
+const heavyUpAndRight = isWindows ? '\u2514' : '┗';
+const heavyUpAndRightLong = heavyUpAndRight + heavyHorizontal;
+const heavyVerticalAndRight = isWindows ? '\u251C' : '┣';
+const heavyVerticalAndRightLong = heavyVerticalAndRight + heavyHorizontal;
+const heavyDownAndHorizontal = isWindows ? '\u252C' : '┳';
 
 class CriticalRequestChains extends Formatter {
 
@@ -149,11 +159,7 @@ class CriticalRequestChains extends Formatter {
 
         // If the parent is the last child then don't drop the vertical bar.
         const ancestorTreeMarker = treeMarkers.reduce((markers, marker) => {
-          if (os.platform() == 'win32') {
-            return markers + (marker ? '\u2502 ' : '  ');
-          } else {
-            return markers + (marker ? '┃ ' : '  ');
-          }
+          return markers + (marker ? heavyVertical : '  ');
         }, '');
 
         // Copy the tree markers so that we don't change by reference.
@@ -164,16 +170,9 @@ class CriticalRequestChains extends Formatter {
 
         // Create the appropriate tree marker based on the depth of this
         // node as well as whether or not it has children and is itself the last child.
-        const treeMarker = ancestorTreeMarker;
-        if (os.platform() == 'win32') {
-          const treeMarker = ancestorTreeMarker +
-            (isLastChild ? '\u2514\u2500' : '\u251C\u2500') +
-            (hasChildren ? '\u252C' : '\u2500');
-        } else {
-          const treeMarker = ancestorTreeMarker +
-            (isLastChild ? '┗━' : '┣━') +
-            (hasChildren ? '┳' : '━');
-        }
+        const treeMarker = ancestorTreeMarker +
+            (isLastChild ? heavyUpAndRightLong : heavyVerticalAndRightLong) +
+            (hasChildren ? heavyDownAndHorizontal : heavyHorizontal);
 
         const parsedURL = CriticalRequestChains.parseURL(node[id].request.url);
 

--- a/lighthouse-core/formatters/critical-request-chains.js
+++ b/lighthouse-core/formatters/critical-request-chains.js
@@ -22,6 +22,7 @@ const path = require('path');
 const fs = require('fs');
 const Formatter = require('./formatter');
 const html = fs.readFileSync(path.join(__dirname, 'partials/critical-request-chains.html'), 'utf8');
+const os = require('os');
 
 class CriticalRequestChains extends Formatter {
 
@@ -148,7 +149,11 @@ class CriticalRequestChains extends Formatter {
 
         // If the parent is the last child then don't drop the vertical bar.
         const ancestorTreeMarker = treeMarkers.reduce((markers, marker) => {
-          return markers + (marker ? '┃ ' : '  ');
+          if (os.platform() == 'win32') {
+            return markers + (marker ? '\u2502 ' : '  ');
+          } else {
+            return markers + (marker ? '┃ ' : '  ');
+          }
         }, '');
 
         // Copy the tree markers so that we don't change by reference.
@@ -159,9 +164,16 @@ class CriticalRequestChains extends Formatter {
 
         // Create the appropriate tree marker based on the depth of this
         // node as well as whether or not it has children and is itself the last child.
-        const treeMarker = ancestorTreeMarker +
+        const treeMarker = ancestorTreeMarker;
+        if (os.platform() == 'win32') {
+          const treeMarker = ancestorTreeMarker +
+            (isLastChild ? '\u2514\u2500' : '\u251C\u2500') +
+            (hasChildren ? '\u252C' : '\u2500');
+        } else {
+          const treeMarker = ancestorTreeMarker +
             (isLastChild ? '┗━' : '┣━') +
             (hasChildren ? '┳' : '━');
+        }
 
         const parsedURL = CriticalRequestChains.parseURL(node[id].request.url);
 

--- a/lighthouse-core/lib/log.js
+++ b/lighthouse-core/lib/log.js
@@ -32,13 +32,7 @@ const colors = {
 };
 
 // whitelist non-red/yellow colors for debug()
-if (isBrowser) {
-  const debugBrowser = require('debug/browser');
-  debugBrowser.colors = [colors.cyan, colors.green, colors.blue, colors.magenta];
-} else {
-  const debugNode = require('debug/node');
-  debugNode.colors = [colors.cyan, colors.green, colors.blue, colors.magenta];
-}
+debug.colors = [colors.cyan, colors.green, colors.blue, colors.magenta];
 
 class Emitter extends EventEmitter {
   /**
@@ -77,9 +71,11 @@ class Log {
       log = debug(title);
       loggersByTitle[title] = log;
       // errors with red, warnings with yellow.
-      // eslint-disable-next-line no-nested-ternary
-      log.color = title.endsWith('error') ? colors.red :
-          title.endsWith('warn') ? colors.yellow : undefined;
+      if (title.endsWith('error')) {
+        log.color = colors.red;
+      } else if (title.endsWith('warn')) {
+        log.color = colors.yellow;
+      }
     }
     return log;
   }

